relentless/simulate/lammps.py

```diff
@@ -471,7 +479,7 @@ def to_commands(self, sim):
         cmds = ['minimize {etol} {ftol} {maxiter} {maxeval}'.format(etol=self.energy_tolerance,
                                                                     ftol=self.force_tolerance,
                                                                     maxiter=self.max_iterations,
-                                                                    maxeval=100*self.max_iterations)]
+                                                                    maxeval=self.options['max_evaluations'])]
```

Comment:
Do we want to give `max_evaluations` a default of 100 so that the user does not necessarily need to specify it? This would be the "typical" behavior in lammps. It would be done above like:
```
if 'max_evaluations' not in self.options:
    self.options['max_evaluations'] = 100
```
rather than raising a `KeyError`. I'm OK if you prefer this to be an explicit option.